bundles/framework/admin-announcements/view/AnnouncementsHelper.js

```diff
@@ -0,0 +1,87 @@
+export const announcementsHelper = () => {
+    const getAdminAnnouncements = (handler) => {
+        if (typeof handler !== 'function') {
+            return;
+        }
+
+        jQuery.ajax({
+            type: 'GET',
+            dataType: 'json',
+            url: Oskari.urls.getRoute('GetAdminAnnouncements'),
+            success: function (pResp) {
+                handler(null, pResp);
+            },
+            error: function (jqXHR, textStatus) {
+                handler('Error', []);
+            }
+        });
+    };
+
+    const saveAnnouncement = (data, handler) => {
+        if (typeof handler !== 'function') {
+            return;
+        }
+        jQuery.ajax({
+            type: 'POST',
+            dataType: 'json',
+            data: {
+                params: JSON.stringify(data)
```

Comment:
This writes the "data" as a serialized JSON format to a parameter named "params"? Wouldn't it be easier and make more sense to just send the announcement as multiple parameters OR have the JSON as payload in the POST body instead of this?